src/Cake.Common/Tools/VSWhere/VSWhereAliases.cs

```diff
@@ -0,0 +1,246 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Cake.Common.Tools.VSWhere.All;
+using Cake.Common.Tools.VSWhere.Latest;
+using Cake.Common.Tools.VSWhere.Legacy;
+using Cake.Common.Tools.VSWhere.Product;
+using Cake.Core;
+using Cake.Core.Annotations;
+
+namespace Cake.Common.Tools.VSWhere
+{
+    /// <summary>
+    /// Contains functionality related to executing VSWhere tool.
```

Comment:
I think we should have information here about requiring the installation of a tool in order for this to work, unless I have missed something.  Similar to what we have done here:

![image](https://cloud.githubusercontent.com/assets/1271146/23920639/6632989e-08f3-11e7-9c13-5f853a12554a.png)